django_feature_flip/factories.py

```diff
@@ -9,3 +10,10 @@ class Meta:
 
     name = factory.Faker('name')
     totally_enabled = False
+
+
+class ActorFactory(factory.django.DjangoModelFactory):
+    class Meta:
+        model = 'django_feature_flip.Actor'
```

Comment:
why are you define the factory model in that way? Can't you just import the models and access to Actor class?

Reply:
Good call, I just followed the docs, but agree, that is a better approach 👍